README.md

```diff
@@ -21,33 +21,39 @@ From the OTP directory:
 1. `mbta/update_pbf.sh` - updates OpenStreetMap data
 1. `mbta/build.sh`
 
-If you want to test with your own local GTFS changes, simply put a copy of your GTFS file (the
-Google output) in `var/MBTA_GTFS.zip` and re-run the `build.sh` script.
+If you want to test with your own local GTFS changes, put a copy of your GTFS file (if you've built
+one with [`gtfs_creator`](https://github.com/mbta/gtfs_creator), it will be written to
+`<gtfs_creator path>/output/google/MBTA_GTFS.zip`) in `var/MBTA_GTFS.gtfs.zip` and re-run the
+`build.sh` script.
 
 ## Run Locally
 With all of that setup done, you should be able to run `mbta/server.sh`. This will start your local
 OTP instance and, when ready, print a message saying that the web server is ready and listening.
 
-Open a browser pointing to `localhost:8080`, and you'll have a bare-bones web interface to OTP where
+Open a browser pointing to [`localhost:8080`](http://localhost:8080), and you'll have a bare-bones web interface to OTP where
 you can try out trip plans.
 
 ## Debugging
-If you need to use a debugger, you can run OTP through IntelliJ. To add a new debug configuration,
-go to `Edit Configurations` and select the Application template
-([see docs](https://www.jetbrains.com/help/idea/run-debug-configuration.html#createExplicitly)).
-
-Populate the fields as follows:
-* Java version: Java 17 (browse to your asdf install for exact parity with the version used by the
-build scripts, though any Java 17 JDK should be fine)
-* Main class: `org.opentripplanner.standalone.OTPMain`
-* Program arguments: `--load var/`
-* Working directory: `<path to your OTP repo>`
-
-Once the config is created, you can run it normally or as a debugger. If you run into any issues
+If you need to use a debugger, you can run OTP through IntelliJ.
+
+1. Start by adding a new debug configuration,
+go to `Edit Configurations` and create a new [Application template, following the IntelliJ
+docs](https://www.jetbrains.com/help/idea/run-debug-configuration.html#createExplicitly) using the
+values provided below.
+1. Populate the fields as follows:
+   * Java version: Java 17 (browse to your asdf install, usually `~/.asdf/installs/java/<version>`,
+   for exact parity with the version used by the build scripts, though any Java 17 JDK is fine)
+   * Main class: `org.opentripplanner.standalone.OTPMain`
+   * Program arguments: `--load var/`
+   * Working directory: `<path to your OTP repo>`
```

Comment:
**suggestion(non-blocking)**: Personally, I think associating this with the parent is clearer. But I'd defer that decision to you 👍🏻 

```suggestion
1. Start by adding a new debug configuration,
go to `Edit Configurations` and create a new [Application template, following the IntelliJ
docs](https://www.jetbrains.com/help/idea/run-debug-configuration.html#createExplicitly) using the
values provided below.
  1. Populate the fields as follows:
     * Java version: Java 17 (browse to your asdf install, usually `~/.asdf/installs/java/<version>`,
     for exact parity with the version used by the build scripts, though any Java 17 JDK is fine)
     * Main class: `org.opentripplanner.standalone.OTPMain`
     * Program arguments: `--load var/`
     * Working directory: `<path to your OTP repo>`
```

Reply:
Made the change but removed the extra numbered list 👍